### PR TITLE
Add HoneyBadger and New Relic execution options

### PR DESCRIPTION
**Pull request by n-hebert**
These are optional, so it should only need to be a v3.0.1 tag here.

# Testing Required?
Yes, as part of ~https://github.com/rewindio/rewind-api/pull/303~ https://github.com/rewindio/shopify-sync-webapp/pull/442/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.swp
```
Comment (davegallant) [inline .gitignore]: Alternatively: `set noswapfile` :)

File: .github/workflows/deploy-env.yml [hunk 2]
```diff
@@ -11,17 +11,43 @@ on:
...
+        description: "The AWS Secret Access Key you would like to deploy with (e.g. prod or staging)"
         required: true
       DEPLOY_FAILURES_SLACK_WEBHOOK_URL:
+        description: "The channel to message in on failures"
         required: true
       DEPLOY_SUCCESS_SLACK_WEBHOOK_URL:
+        description: "The channel to message in on successes"
         required: true
       LOOKUP_USER_EMAIL_SLACK_TOKEN:
+        description: "A token to look up users by email - not guaranteed to work for Orgs"
```
Comment (davegallant) [inline .github/workflows/deploy-env.yml]: This only works if you have your GItHub email matches what is in Slack _and_ your settings do not enable `Keep my email addresses private`.

File: .github/workflows/deploy-env.yml [hunk 3]
```diff
@@ -11,17 +11,43 @@ on:
...
+        type: boolean
     secrets:
       EB_AWS_ACCESS_KEY_ID:
+        description: "The AWS Access Key ID you would like to deploy with (e.g. prod or staging)"
         required: true
       EB_AWS_SECRET_ACCESS_KEY:
+        description: "The AWS Secret Access Key you would like to deploy with (e.g. prod or staging)"
         required: true
       DEPLOY_FAILURES_SLACK_WEBHOOK_URL:
+        description: "The channel to message in on failures"
```
Comment (davegallant) [inline .github/workflows/deploy-env.yml]: Maybe something like:

The Slack webhook url that maps to a Slack channel. This channel will be notified after a successful deploy.

Might also be worth mentioning that it should be secret.
i.e. https://www.darkreading.com/cloud/slack-s-incoming-webhooks-can-be-weaponized-in-phishing-attacks

File: .github/workflows/deploy-env.yml (hunk 2 above)
Comment (davegallant) [reply .github/workflows/deploy-env.yml]: It does fail gracefully though.

File: .github/workflows/deploy-env.yml (hunk 3 above)
Comment (n-hebert) [reply .github/workflows/deploy-env.yml]: Thanks! I'll update

File: .gitignore (hunk 1 above)
Comment (n-hebert) [reply .gitignore]: Good point! I do occasionally recover and I definitely need the help remembering I have the file open.

**Review by davegallant — APPROVED** [review]

**Comment by n-hebert** [comment]
image

Honeybadger kinda works! 
I wonder why it says production. I'll look into it.

**Comment by n-hebert** [comment]
It's just the default.
https://github.com/honeybadger-io/github-notify-deploy-action

I wonder what might be the best thing to put there, especially considering workers, etc.

**Pull request merged**